sdk/textanalytics/Azure.AI.TextAnalytics/src/DetectedLanguage.cs

```diff
@@ -20,6 +20,14 @@ internal DetectedLanguage(DetectedLanguageInternal language, IList<TextAnalytics
             Warnings = new ReadOnlyCollection<TextAnalyticsWarning>(warnings);
```

Comment:
Note to self:  update so that this delegates to the component constructor and removes duplication.